scripts/firedrake-zenodo

```diff
@@ -23,6 +24,7 @@ descriptions = OrderedDict([
 projects = dict(
     [("firedrake", "firedrakeproject"),
      ("PyOP2", "OP2"),
+     ("SLOPE", "coneoproject"),
      ("tsfc", "firedrakeproject"),
      ("COFFEE", "coneoproject"),
      ("ufl", "firedrakeproject"),
```

Comment:
My bad. I thought for a moment this was handled (so we are not zenodo-ing firedrake-adjoint? I only now notice it's not in the ``components`` dictionary). When I tried it, I was in the repo with SLOPE installed :)